### DISCUSSION TITLE
Apply settings to all migrations

### DISCUSSION THREAD

**Discussion by astellingwerf** [renovatebot/renovate#16609] [Suggest an Idea]
What I don't see in current documentation or in #16359, is a section where users can put properties that apply to all migration MRs, a bit like [`major`](https://docs.renovatebot.com/configuration-options/#major) for example.

I have no clue on how Migrations are different beasts than package updates or onboarding PRs, but I'd expect I can set settings like `automerge` on them.

**Comment by PhilipAbed** [comment]
@HonkingGoose can you  please check this?

**Reply by HonkingGoose** [reply]
We don't have any documentation about "where users can put properties that apply to all migration pull requests, like `major` for example".

I searched in our docs site with the `migration` keyword, with these (somewhat) relevant results:

- [`configMigration`](https://docs.renovatebot.com/configuration-options/#configmigration)
- [`autoReplaceStringTemplate`](https://docs.renovatebot.com/configuration-options/#autoreplacestringtemplate)
- [`allowPostUpgradeCommandTemplating`](https://docs.renovatebot.com/self-hosted-configuration/#allowpostupgradecommandtemplating)

The `migration` keyword also shows up in our [Gitlab bot security, recommended migration](https://docs.renovatebot.com/gitlab-bot-security/#recommended-migration), but that's for a different migration.

---

> I have no clue on how Migrations are different beasts than package updates or onboarding PRs, but I'd expect I can set settings like `automerge` on them.

@rarkins can you answer this? You should probably also decide what kind of settings Renovate should support for `configMigration`.
